### ISSUE TITLE
VeraCrypt with no additional software?

### ISSUE THREAD

**Issue by torinthiel** [freedomofpress/securedrop-docs#515]
## Describe the change
Instructions for setting up Export Device say that 'VeraCrypt-encrypted media can be opened in the Tails operating system and on common Linux distributions without installing additional software'. While this is technically true for Tails (as VeraCrypt comes pre-installed), I think it's still a separate package that needs to be installed, even if it comes from distro's native package set.

## How will this impact [SecureDrop users](https://github.com/freedomofpress/securedrop-ux/wiki/Users)?
Users would know not to expect automatic mounting of VeraCrypt images under Linux.

**Comment by eloquence** [comment]
@torinthiel Fair point, it's definitely distro-dependent beyond Tails. Clarifying the wording here is probably not controversial - would you like to take a stab at it yourself?
